src/agent/src/rpc.rs

```diff
@@ -2481,6 +2481,26 @@ mod tests {
         // normally this module should eixsts...
         m.name = "bridge".to_string();
         let result = load_kernel_module(&m);
+
+        // Skip test if loading kernel modules is not permitted
+        // or kernel module is not found
+        if let Err(e) = &result {
+            let error_string = format!("{:?}", e);
+            // Let's print out the error message first
+            println!("DEBUG: error: {}", error_string);
+            if error_string.contains("Operation not permitted")
```

Comment:
Would be possible to check the error number instead of the message? Less error prone IMO.

Reply:
The error message is:

```
modprobe: FATAL: Module bridge not found in directory /lib/modules/6.8.0-85-generic
```